### PR TITLE
added reply to self message issue:(#40)

### PR DESCRIPTION
**Pull request by Zain-ul-din**
### APIS

To enable a reply to self message user needs to set `REPLY_SELF_ENABLED=true` in `.env` file.
Usage has been added in `readme.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ Whatsapp bot that uses OpenAI's GPT & DALLE to respond to user inputs.
 
 ```
 OPENAI_API_KEY=put_your_key_here
-
 PREFIX_ENABLED=false
+REPLY_SELF=false
```
Comment (marcomafessolli) [inline README.md:22]: Maybe?

```suggestion
REPLY_SELF_ENABLED=false
```

**Review by Zain-ul-din — COMMENTED** [review]
Tested by my side

File: src/index.ts [hunk 2]
```diff
@@ -36,29 +65,16 @@ const start = async () => {
 
     // Whatsapp message
     client.on("message", async (message: any) => {
-        const messageString = message.body
-        if (messageString.length == 0) return
+        const messageString = message.body;
```
Comment (marcomafessolli) [inline src/index.ts:68]: I think this is no longer necessary

**Review by marcomafessolli — COMMENTED** [review]
Works on my end as well!

File: src/index.ts (hunk 2 above)
Comment (Zain-ul-din) [reply src/index.ts:68]: yep!

**Comment by Zain-ul-din** [comment]
when you will merge it?

**Comment by Re1da** [comment]
`

**Comment by Re1da** [comment]
انا  رضا ديكور

**Comment by Re1da** [comment]
اعتقد اني قد عشت بداخل عقلي أكثر مما يجب

**Comment by navopw** [comment]
> when you will merge it?
> 
> 

Whats the reason someone will rnable or disable self reply?

I see no use case.

**Pull request merged**